src/services/hackerNews.js

```diff
@@ -0,0 +1,51 @@
+import http from '../utils/http';
+import API from '../constants/api';
+
+/**
+ * @param {String} newsType
+ * @param {Number} newsId
+ * Fetch news data from the api.
+ */
+export async function getNews(newsType, newsId = null) {
+  if (newsId === null) {
+    const filteredNewsType = matchNewsTypeToUrl(newsType);
+    const apiData = await http.get(filteredNewsType);
+
+    return apiData.data;
+  } else if (newsId !== null) {
+    const apiData = await http.get(`/item/${newsId}.json`);
+
+    return apiData;
+  }
+}
+
+/**
+ * @param {*} newsType
+ * @param {*} commentId
+ * Fetch comments for specific news.
+ */
+export async function getComments(newsType, commentId) {
+  if (newsType === 'comments') {
```

Comment:
Can newsType be different other than comments. Function name and implementation doesn't match